### ISSUE TITLE
Solved: yarn build issues related to webpack

### ISSUE THREAD

**Issue by LCinHK** [misskey-dev/misskey#7651]
## 💡 Summary
I am trying to build misskey on my windows local environment and some issues related to webpack appeared
## 🙂 Expected Behavior

After `yarn build` it should work

## ☹️ Actual Behavior
```
[webpack-cli] TypeError: The 'compilation' argument must be an instance of Compilation
    at Function.getCompilationHooks (C:\git-work\misskey\node_modules\webpack\lib\NormalModule.js:195:10)
    at C:\git-work\misskey\node_modules\vue-loader\dist\pluginWebpack5.js:45:26
    at Hook.eval (eval at create (C:\Git-work\misskey\node_modules\tapable\lib\HookCodeFactory.js:19:10), <anonymous>:102:1)
    at Hook.CALL_DELEGATE [as _call] (C:\Git-work\misskey\node_modules\tapable\lib\Hook.js:14:14)
    at Compiler.newCompilation (C:\Git-work\misskey\node_modules\webpack\lib\Compiler.js:1044:26)
    at C:\Git-work\misskey\node_modules\webpack\lib\Compiler.js:1088:29
    at Hook.eval [as callAsync] (eval at create (C:\Git-work\misskey\node_modules\tapable\lib\HookCodeFactory.js:33:10), <anonymous>:6:1)
    at Hook.CALL_ASYNC_DELEGATE [as _callAsync] (C:\Git-work\misskey\node_modules\tapable\lib\Hook.js:18:14)
    at Compiler.compile (C:\Git-work\misskey\node_modules\webpack\lib\Compiler.js:1083:28)
    at C:\Git-work\misskey\node_modules\webpack\lib\Compiler.js:508:12
error Command failed with exit code 2.
```
## 📝 Steps to Reproduce

1. `git pull` the latest misskey
2. try to yarn install, which appears to be fine
3. `yarn build` under environment production gives error

## 📌 Environment

```
Windows 11
node v16.6.2

trying to install misskey@12.88.0
```

**Comment by Johann150** [comment]
I think you should try with an older Node version that is listed as supported in the setup file (version 12 or 14).

https://github.com/misskey-dev/misskey/blob/cf0a5d5735a0eb18e21c07c09e7261c02ec84479/docs/setup.en.md#L26

Maybe also `yarn clean` or `yarn cleanall` can be helpful, see [docs/setup.en.md](https://github.com/misskey-dev/misskey/blob/master/docs/setup.en.md).

**Comment by LCinHK** [comment]
> Maybe also `yarn clean` or `yarn cleanall` can be helpful, see [docs/setup.en.md](https://github.com/misskey-dev/misskey/blob/master/docs/setup.en.md).
## Solved: `yarn cleanall` worked (with latest v12.89 version)
Thank you!
